Libraries/LibWeb/Crypto/SubtleCrypto.cpp

```diff
@@ -611,7 +611,7 @@ JS::ThrowCompletionOr<GC::Ref<WebIDL::Promise>> SubtleCrypto::derive_bits(Algori
         // 6. If the following steps or referenced procedures say to throw an error, reject promise with the returned error and then terminate the algorithm.
 
         // 7. If the name member of normalizedAlgorithm is not equal to the name attribute of the [[algorithm]] internal slot of baseKey then throw an InvalidAccessError.
-        if (normalized_algorithm.parameter->name != base_key->algorithm_name()) {
```

Comment:
While this passes more subtests, it goes against the spec. A bug report in WPT or the spec would be great :) 

https://www.w3.org/TR/WebCryptoAPI/#conformance
> Unless otherwise stated, string comparisons are done in a [case-sensitive](https://www.w3.org/TR/WebCryptoAPI/#case-sensitive) manner

Reply:
I've dropped the commit for now since it's not that important. I've also noticed there are some places where existing code is not fully compliant, will do a separate PR.

I've never submitted a spec bug, so I'll take my time. If anyone wants to do it, feel free.

Reply:
I have made a separate PR for this: https://github.com/LadybirdBrowser/ladybird/pull/2614.